test/framework/clustervalidator.go

```diff
@@ -75,6 +79,7 @@ func NewClusterValidator(opts ...ClusterValidatorOpts) *ClusterValidator {
 		ClusterOpts: clusterOpts{},
 		validations: []retriableValidation{},
 	}
+	cv.baseValidations()
```

Comment:
Why do we need to have base validations in the constructor vs registering them when they're needed with `WithWorkloadClusterValidations`?